tests/autoskip/arg-change.earth

```diff
@@ -0,0 +1,10 @@
+VERSION --build-auto-skip 0.8
+
+ARG VERSION=3.18.6
```

Comment:
I worry that someone might be tempted to bump this version (thinking it's good to keep up to date); however it won't be obvious that it needs to stay in sync with the corresponding test in the Earthfile.

I have two ideas:

1. change this to `ARG VERSION=PLACEHOLDER_FOR_DEFAULT_VALUE`, and then in the `test-arg-change` target, do a `sed -i "s/PLACEHOLDER_FOR_DEFAULT_VALUE/3.18.6"`.

or

2. instead of changing the versions, just change the image, e.g.

```
ARG IMG=alpine
FROM $IMG
```

then `sed -i "s/alpine/busybox/g" Earthfile`, and then they will just always use the latest.

Reply:
Good point. Does a comment or special Renovate ignore flag sound fair? @idodod do you know how to achieve the latter?

Reply:
To elaborate, it would be useful to have a way to pin versions without the extra effort.

Reply:
We don't use Renovate (yet) to upgrade the images in Earthfiles but once we do it shouldn't be a problem for it to ignore this specific image version. 

To Alex's point though, Someone (like me:)) might still feel the urge to upgrade it manually. Maybe using a digest might make it clearer that it shouldn't be upgraded?

Reply:
> it would be useful to have a way to pin versions

my second idea of using alpine vs busybox would allow this.

```
ARG IMG=alpine:3.19
FROM $IMG
```

then update this:

```
DO --pass-args tests+RUN_EARTHLY --extra_args="--auto-skip-db-path=test.db" --target=+expand-args-from --output_contains="IMG=alpine"
RUN sed -i "s/alpine:.*$/busybox:1.36.1/g" Earthfile
DO --pass-args tests+RUN_EARTHLY --extra_args="--auto-skip-db-path=test.db" --target=+expand-args-from --output_contains="IMG=busybox"
```

Reply:
update: my incorrectly understood the `output_contains="VERSION_ID=3.18.6"` part of the test, this requires `cat /etc/os-release`.

Mike and I chatted about this, and instead we can go with the first idea where the version is only specified in `tests/autoskip/Earthfile`, and will use sed to update the value in the `tests/autoskip/arg-change.earth` file